app/views/registration_wizard/aso_unavailable.html.erb

```diff
@@ -5,9 +5,16 @@
     wizard: @wizard,
     ) do
 %>
-  <h1 class="govuk-heading-xl">You cannot register for the Early headship coaching offer</h1>
+  <%
```

Comment:
Can this not go somewhere shared? i was trying to find the page

Reply:
What in particular would go somewhere shared? You're probably right but I want to be clear what you mean.